public/images/svg/travis-console.svg

```diff
@@ -0,0 +1,231 @@
+	<path class="st2" d="M253.758,236.466c0,4.283-3.504,7.787-7.787,7.787H32.961c-4.283,0-7.787-3.504-7.787-7.787V114.048
+		c0-4.283,0-11.291,0-15.573v-3.641c0-4.283,3.504-7.787,7.787-7.787h213.01c4.283,0,7.787,3.504,7.787,7.787V236.466z"/>
+</g>
+<g class="st3">
+	<pattern  id="SVGID_2_" xlink:href="#Blue_waves" patternTransform="matrix(1.343 0 0 1 2274.2332 -629.0682)"  class="st3">
+	</pattern>
+	<path class="st4" d="M253.758,236.466c0,4.283-3.504,7.787-7.787,7.787H32.961c-4.283,0-7.787-3.504-7.787-7.787V114.048
+		c0-4.283,0-11.291,0-15.573v-3.641c0-4.283,3.504-7.787,7.787-7.787h213.01c4.283,0,7.787,3.504,7.787,7.787V236.466z"/>
+</g>
+<g>
+	<g>
+		<path class="st5" d="M284.121,214.082c0,4.578-3.746,8.324-8.324,8.324H62.233c-4.578,0-8.324-3.746-8.324-8.324v-77.369
+			c0-4.578,0-12.069,0-16.647v-9.774c0-1.249,1.012-2.261,2.261-2.261h225.69c1.249,0,2.261,1.012,2.261,2.261V214.082z"/>
+	</g>
+	<g>
+		<path class="st6" d="M53.909,74.382c0-4.578,3.746-8.324,8.324-8.324h213.565c4.578,0,8.324,3.746,8.324,8.324v77.369
+			c0,4.578,0,4.578,0,0v-45.98c0,1.249-1.012,2.261-2.261,2.261H56.17c-1.249,0-2.261-1.012-2.261-2.261V74.382z"/>
+	</g>
+	<g>
+		<path class="st7" d="M284.121,212.674c0,5.353-4.379,9.732-9.732,9.732H63.641c-5.353,0-9.732-4.379-9.732-9.732V103.619
+			c0-5.353,0-14.111,0-19.464V75.79c0-5.353,4.379-9.732,9.732-9.732h210.749c5.353,0,9.732,4.379,9.732,9.732V212.674z"/>
+	</g>
+	<line class="st8" x1="284.121" y1="108.031" x2="53.909" y2="108.031"/>
+	<circle class="st9" cx="78.635" cy="87.475" r="7.872"/>
+	<circle class="st10" cx="103.979" cy="87.475" r="7.872"/>
+	<circle class="st11" cx="129.323" cy="87.475" r="7.872"/>
+</g>
+<text transform="matrix(1 0 0 1 70.7621 159.6499)" class="st12 st13 st14">Travis whatsup</text>
+<path class="st11" d="M293.651,32.947v-0.008c-6.934-0.791-12.427-6.298-13.204-13.235h-0.007c-0.004-0.052-0.045-0.093-0.098-0.093
+	s-0.093,0.042-0.098,0.093h-0.006c-0.778,6.945-6.281,12.456-13.225,13.239v0.005c-0.052,0.004-0.093,0.045-0.093,0.098
+	c0,0.053,0.042,0.093,0.093,0.098v0.005c6.938,0.782,12.437,6.284,13.222,13.22h0.01c0.004,0.052,0.045,0.093,0.098,0.093
+	c0.053,0,0.093-0.042,0.098-0.093h0.007c0.785-6.928,6.274-12.426,13.202-13.216v-0.008c0.052-0.004,0.093-0.045,0.093-0.098
+	C293.744,32.992,293.702,32.951,293.651,32.947z"/>
+<path class="st15" d="M70.274,270.892v-0.005c-4.078-0.466-7.309-3.704-7.766-7.785h-0.004c-0.002-0.03-0.026-0.055-0.057-0.055
+	c-0.031,0-0.055,0.025-0.057,0.055h-0.003c-0.458,4.085-3.694,7.326-7.779,7.787v0.003c-0.03,0.002-0.055,0.026-0.055,0.057
+	c0,0.031,0.025,0.055,0.055,0.057v0.003c4.08,0.46,7.315,3.696,7.777,7.775h0.006c0.002,0.03,0.026,0.055,0.057,0.055
+	c0.031,0,0.055-0.025,0.057-0.055h0.004c0.462-4.075,3.69-7.308,7.765-7.773v-0.005c0.03-0.002,0.055-0.026,0.055-0.057
+	C70.329,270.918,70.304,270.894,70.274,270.892z"/>
+<g>
+	
+		<ellipse transform="matrix(0.149 -0.9888 0.9888 0.149 157.2976 416.1463)" class="st16" cx="320.416" cy="116.689" rx="5.547" ry="5.547"/>
+</g>
+</svg>
```

Comment:
This image looks a lot like a "Mastercard" to me 😄

Reply:
Ohhhh yeah, it totally does XD haha